lib/Applify.pm

```diff
@@ -175,13 +180,21 @@ OPTION:
 
   print "Usage:\n";
 
-  if (%{$self->{subcommands} || {}}) {
-    my $subcmds = [sort { $a->{name} cmp $b->{name} } values %{$self->{subcommands}}];
-    my ($width) = sort { $b <=> $a } map { length($_->{name}) } @$subcmds;
-    print "\n    ", File::Basename::basename($0), " [command] [options]\n";
-    print "\nCommands:\n";
-    printf("    %-${width}s  %s\n", @{$_}{'name', 'desc'}) for @$subcmds;
-    print "\nOptions:\n";
+  if ( $app_help  ) {
```

Comment:
I think we can avoid nested if's:

```
if ($self->{subcommand}) { ... } # a subcommand specified
elsif (%{$self->{subcommands} || {}}) { ... } # no subcommand, but got sub commands
else { ... } # no subcommand, nor subcommands
```

Reply:
Sure. I've also regularized the output for all of the conditional branches (previously "Options" was only printed for top-level help with subcommands)  Do you want the synopsis printed for the subcommand help, or just for the top-level help?

Reply:
Good question. I'm not a user of the sub command feature, so I'm not sure. I think it should just be included for the top level help, but I'm fine either way.

You decide 👍

Reply:
Ok; I agree that it should just be for the top level.